app_server/app/controllers/SubscriptionController.js

```diff
@@ -136,6 +141,11 @@ Class.getSubscribers = function(request, reply) {
       return reply(result);
     });
 };
+Class.getSelfSubscribers = function(request, reply) {
+  logger.debug('Get list of self subscribers');
```

Comment:
"self subscribers" isn't really too self-explanatory.
Also, add a line break between this and the previous.